docs/database-functionalities/manage-user-privileges.md

```diff
@@ -56,8 +57,12 @@ We could naturally cluster those privileges into groups:
   * Privilege to create and delete data (`CREATE`, `DELETE`, `REMOVE`)
   * Privilege to index data (`INDEX`)
   * Privilege to obtain statistics and information from Memgraph (`STATS`)
-  * Privilege to use data streaming (`STREAM`)
```

Comment:
It is good we have this, when we add kafka integration again, we just have to revert these lines :smile:

Reply:
@antonio2368 Should all of the changes in this PR be applied to both versions of Memgraph (1.3 & 1.4)?

Reply:
I wouldn't do that, because it didn't work in those versions... :smile: